OpenOversight/app/templates/browse.html

```diff
@@ -20,16 +20,24 @@ <h2>{{ department.name }}
             </a>
         {% endif %}
        </h2>
-          <p>
+
+       <div>
+         <i class="dept-{{department.id}}">Officers Updated: {{department.latest_officer_update()}}</i>
```

Comment:
This displays as `None` if there's no date, could we use "No data" in those cases instead?

Reply:
Yup, changed!